pkg/providers/instance/instance.go

```diff
@@ -415,7 +415,14 @@ func (p *Provider) launchInstance(
 	return resp, instanceType, nil
 }
 
+// nolint:gocyclo
 func (p *Provider) handleResponseErrors(ctx context.Context, instanceType *corecloudprovider.InstanceType, zone, capacityType string, err error) error {
+	if sdkerrors.LowPriorityQuotaHasBeenReached(err) {
+		// Mark in cache that spot quota has been reached for this subscription
+		p.unavailableOfferings.Put(sdkerrors.LowPriorityQuotaExceededTerm, struct{}{}, SubscriptionQuotaReachedTTL)
```

Comment:
See elsewhere on Put. I also slightly dislike the use of the error term as key, but this is less important (and best decided within unavailableOffereings implementation)

Reply:
Agree removed